recipes/fermi-lite/meta.yaml

```diff
@@ -6,11 +6,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/lh3/fermi-lite/archive/v{{ version }}.tar.gz
+  url: https://github.com/mr-c/fermi-lite/archive/v{{ version }}.tar.gz
```

Comment:
Do we need to change the source to a fork of the repo?

Reply:
I wish/hope lh3 merges in the changes from mr-c so we don't have to.

See the first link on this PR's description with my comment on fermi-lite's non-x86 PR ☝🏻 .

Reply:
Would you be open to keeping the source as the official repo but including a patch in Bioconda as alternative given this is a small change to the codebase?

Reply:
Since I'm not exactly an expert with Bioconda recipes / preferences, it might worthwhile have someone from the @bioconda/review team make a suggestion here

Reply:
That makes sense, sure! I'll change it accordingly.

Reply:
I would agree that patch would be preferable. I'd further recommend it be conditioned on a `# [aarch64]` selector, so only that only that build variant applies it.